tests/tests.py

```diff
@@ -564,7 +592,7 @@ def test_process_with_vpc_config(self):
         wanted = ["dbhost", "dbport"]
         output_names = cfn_template['Outputs'].keys()
         output_names.sort()
-        compare(output_names, wanted)
+        compare(wanted, output_names)
```

Comment:
I'm not sure if there's a proper order for wanted vs expected (the docs just say one and two I think) but why did we swap these parameters around?